app/src/main/java/sopt/motivoo/presentation/NetworkErrorFragment.kt

```diff
@@ -0,0 +1,38 @@
+package sopt.motivoo.presentation
+
+import android.os.Bundle
+import android.view.View
+import androidx.activity.OnBackPressedCallback
+import androidx.navigation.fragment.findNavController
+import sopt.motivoo.R
+import sopt.motivoo.databinding.FragmentConnectionErrorBinding
+import sopt.motivoo.util.binding.BindingFragment
+import sopt.motivoo.util.extension.setOnSingleClickListener
+
+class NetworkErrorFragment :
+    BindingFragment<FragmentConnectionErrorBinding>(R.layout.fragment_connection_error) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        backPressed()
```

Comment:
조금 더 명확한 함수명을 사용해도 좋을 것 같아요!
```suggestion
        overrideOnBackPressed()
```